layouts/GeneralLayout.tsx

```diff
@@ -33,8 +33,7 @@ const GeneralLayout = ({ children }: { children: ReactNode }) => {
           className={clsx(
             'flex flex-col min-h-full px-4 pb-2 md:ml-[290px] lg:w-[calc(100%-290px)]',
 
-            pathname != '/' && ' transition-color ease-in-out duration-200',
-            'bg-theme-primary-light dark:bg-inherit'
+            pathname != '/' && 'bg-theme-primary-light dark:bg-inherit'
           )}
```

Comment:
We should also remove the check `pathname != '/' &&`.

```suggestion
          className="flex flex-col min-h-full px-4 pb-2 md:ml-[290px] lg:w-[calc(100%-290px)] bg-theme-primary-light dark:bg-inherit"
```

Reply:
done the suggested changes.